start-goss-servers.sh

```diff
@@ -60,8 +60,13 @@ while true ; do
     sleep 5
 done
 
-# for security reasons we only want to run the servers on the HMN network, which is not connected to open Internet
-ip=$(host "$(hostname).hmn" | grep -Po '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')
+if lsblk --noheadings -o vendor | grep Google; then
```

Comment:
If that command is going to be the general way to check if this is vshasta or not, then I'd recommend adding it to the shared Bash function library (`automated/run-ncn-tests.sh`) as a function like `is_vshasta`, similar to the `is_pit_node` function. That would make it easier to re-use, and if we ever need to change it, we won't have to hunt down every place we use the command.

Reply:
We actually have a better `is_vshasta` check, which is to see if `/etc/google_system` exists, but we boot the metal images, which doesn't have that file, so the method of determining if a node is vshasta or not needs some more consideration I think, rather than what I have here.

Reply:
Either way, with whatever method you arrive at, it seems like we'd want it to be in a common function for easier reuse, right?

Reply:
Yeah, it is not a hard change to make.  I am fixing it up.

Reply:
Fixed.  More relevant info here: https://github.com/Cray-HPE/csm-testing/pull/414#discussion_r1018211533